test/integration_test.py

```diff
@@ -564,7 +564,7 @@ def _get_one_file(self, catalog: CatalogName) -> JSON:
         if config.is_hca_enabled(catalog):
             file_size_facet = 'fileSize'
         elif config.is_anvil_enabled(catalog):
-            file_size_facet = 'size'
+            file_size_facet = 'file_size'
```

Comment:
```suggestion
            file_size_facet = 'files.file_size'
```

I thought we agreed that all organic (as opposed to synthetic) fields are qualified.

Reply:
This test was disabled due to https://github.com/DataBiosphere/azul/issues/4507. If it had been run, it would have failed due to the unqualified name.